app/src/main/res/values-pl/strings.xml

```diff
@@ -0,0 +1,853 @@
+	<string name="draft_prefix">Wersja robocza, ostatni zapis %s</string>
+	<string name="editing_prefix">Edycja, ostatni zapis %s</string>
+	<string name="quantity">Ilość</string>
+	<string name="quantity_hint">Ilość</string>
+	<string name="length">Czas trwania</string>
+	<string name="length_hint">Czas trwania w minutach</string>
+	<string name="no_length">\?</string>
+	<string name="location">Lokalizacja</string>
+	<string name="location_hint">Lokalizacja</string>
+	<string name="no_location">&lt;Brak lokalizacji&gt;</string>
+	<string name="nowhere_in_angle_brackets">&lt;Nigdzie&gt;</string>
+	<string name="incomplete">Niezakończona</string>
+	<string name="noWinStats">Nie licz do statystyk wygranych</string>
+	<string name="comments">Komentarze</string>
+	<string name="send">Wyślij</string>
+	<string name="save">zapisz</string>
+	<string name="done">Zakończone</string>
+	<string name="ok">OK</string>
+	<string name="cancel">Anuluj</string>
+	<string name="discard">Odrzuć</string>
+	<string name="continue_">Kontynuuj</string>
+	<string name="reset">Resetuj</string>
+	<string name="sort">Sortuj</string>
+	<string name="keep_editing">Edytuj</string>
+	<string name="cancelled">Anulowano</string>
+	<string name="exact">Dokładny</string>
+	<string name="exactly">Dokładnie</string>
+	<string name="select_range">Zaznacz przedział</string>
+	<string name="include_missing_values">Uwzględnij gry z brakującymi danymi</string>
+	<string name="filter_description_player_number">Zostają pokazane tylko gry dla <b>całego zakresu</b>graczy. <i>Dokładny</i> znaczy, że pokazane zostaną tylko gry dla danej liczby graczy (np. gry tylko dla 2 graczy).</string>
+	<string name="filter_description_include_missing_play_time">Jeśli <b>uwzględnisz gry z brakującymi danymi</b> filtr pokaże gry z nieznanym czasem gry.</string>
+	<string name="filter_description_include_missing_suggested_age">Jeśli <b>uwzględnisz gry z brakującymi danymi</b> filtr pokaże gry bez sugerowanego minimalnego wieku.</string>
+	<string name="filter_description_include_missing_average_weight">Jeśli <b>uwzględnisz gry z brakującymi danymi</b> filtr pokaże gry bez określonej średniej trudności.</string>
+	<string name="clear">Wyczyść</string>
+	<string name="set">Ustaw</string>
+	<string name="plays">rozgrywek</string>
+	<string name="played">Rozegrane</string>
+	<string name="rated">Ocenione</string>
+	<string name="commented">Skomentowane</string>
+	<string name="rematch">Rewanż</string>
+	<string name="menu_change_game">Zmień grę</string>
+	<string name="title_pick_start_player">Wybierz gracza rozpoczynającego</string>
+	<string name="title_players">Gracze</string>
+	<string name="title_players_with_count">Gracze - %,d</string>
+	<string name="title_location">Lokalizacja</string>
+	<string name="title_locations">Lokalizacje</string>
+	<string name="title_player">Gracz</string>
+	<string name="title_firebase_message">Wiadomość od twórców aplikacji</string>
+	<string name="or">Lub</string>
```

Comment:
It's used in `CollectionStatusFilterDialog.kt` and `PlayStatsFragment.kt`. The case is, when strictly translated it sounds well in 1st place and kind of "well" in 2nd place (and the opposite, when not strictly translated). It can work as it is. But will definitely sound better in Polish if this line is separated into two lines: "or"→pl:"lub" (for 1st place) and "nor"→pl:"ani" (for 2nd place).